scripts/env.sh

```
@@ -73,14 +72,17 @@ BRIDGE_NAME=${BRIDGE_NAME:-br0}
 SKIP_BRIDGE_CONFIG=${SKIP_BRIDGE_CONFIG:-"false"}
 
 # DPF Configuration
+DPF_VERSION="v25.4.0"
```

Shellcheck (0.10.0)

[warning] DPF_VERSION appears unused. Verify use (or export if used externally).

(SC2034)